src/contrib/cluster/Akka.Cluster.Sharding.Tests/ClusterShardingSpec.cs

```diff
@@ -295,7 +295,7 @@ private void CreateCoordinator()
 
             foreach (var typeName in typeNames)
             {
-                var rebalanceEnabled = string.Equals(typeName, "rebalancing", StringComparison.InvariantCultureIgnoreCase);
+                var rebalanceEnabled = string.Equals(typeName, "rebalancing", StringComparison.OrdinalIgnoreCase);
```

Comment:
MS recommends to replace InvariantCultureIgnoreCase to OrdinalIgnoreCase in many cases